x/hub-genesis/keeper/msg_server.go

```diff
@@ -3,6 +3,8 @@ package keeper
 import (
 	"context"
 
+	errorsmod "cosmossdk.io/errors"
+
```

Comment:
just curious, what's this? why the change?
I dont' recall this from when I worked on cosmos over 1 year ago 😄

Reply:
ah I see the deprecation